streams/examples/src/main/java/org/apache/kafka/streams/examples/wordcount/WordCountProcessorDemo.java

```diff
@@ -93,8 +93,6 @@ public void process(final String dummy, final String line) {
                             this.kvStore.put(word, oldValue + 1);
                         }
                     }
-
-                    context.commit();
```

Comment:
https://github.com/apache/kafka/pull/6824#discussion_r424769188

If I were to guess, @mjsax wanted it removed because it isn't necessary and distracts from the most simple form of the example.

Reply:
Ah, I _did_ miss something. Thanks.